Source/Plugins/Core/com.equella.base/src/com/tle/common/settings/standard/SearchSettings.java

```diff
@@ -73,6 +74,7 @@
   @Property(key = "search.defaultsort")
   private String defaultSearchSort;
 
+  @JsonIgnore
```

Comment:
When reading this class by itself it's going to seem a bit out of place to have a stray `@JsonIgnore`. So how about we add a comment:

```suggestion
  @JsonIgnore // The REST endpoints for this was split into two - one for general search settings, and one for filters - so we don't want this included in the general search settings.
```